galera/manifests/server.pp

```diff
@@ -108,13 +109,14 @@
   $service_name          = 'mariadb',
   $service_enable        = true,
   $service_ensure        = 'running',
+  $mysql_root_password   = '',
 )  {
   if $create_mysql_resource {
   warning("DEPRECATED: ::mysql::server should be called manually, please set create_mysql_resource to false and call class ::mysql::server with your config")
 
     $mysql_server_class = { 'mysql::server' => $mysql_server_hash }
```

Comment:
I see, so setting $mysql::server to the empty hash in this line is what creates the resource.